job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/FinalOutputDeliveryHandler.java

```diff
@@ -66,7 +66,8 @@ public FinalOutputDeliveryHandler(Codec codec, JobsApi jobsApi, ExecutionContext
 
 
     @Override
-    public void handleResult(TaskMessage resultMessage) {
+    public void handleResult(T resultMessageInput) {
+        final TaskMessage resultMessage = (TaskMessage) resultMessageInput;
```

Comment:
If this is always a `TaskMessage` then why aren't we just implementing `ResultHandler<TaskMessage>` instead of `ResultHandler<T>`?